jerry-ext/include/jerryscript-ext/module.h

```diff
@@ -18,6 +18,12 @@
 
 #include "jerryscript.h"
 
+#ifdef __cplusplus
+#define JERRYX_EXTERN_C extern "C"
+#else
+#define JERRYX_EXTERN_C
+#endif
```

Comment:
This does not look right here. No.1: it is confusing to have this here right in front of another `#ifdef __cplusplus`. No.2: This is only used by MSVC, so it should be added to the MSVC-specific block, if it is indeed necessary.

Reply:
Yes, it's necessary, as without exntern "C", C++ code include this module.h would take no effect